src/energyplus/ForwardTranslator/ForwardTranslateRoofVegetation.cpp

```diff
@@ -138,6 +144,21 @@ namespace energyplus {
       idfObject.setString(Material_RoofVegetationFields::MoistureDiffusionCalculationMethod, *s);
     }
 
+    // Call the translation of these objects, which has two advantages:
+    // * will not translate them if they are orphaned (=not referencing a material), and,
+    // * makes the order of these objects in the IDF deterministic
+    if (boost::optional<MaterialPropertyMoisturePenetrationDepthSettings> _empd = modelObject.materialPropertyMoisturePenetrationDepthSettings()) {
+      translateAndMapModelObject(_empd.get());
+    }
+
+    if (boost::optional<MaterialPropertyPhaseChange> _phaseChange = modelObject.materialPropertyPhaseChange()) {
+      translateAndMapModelObject(_phaseChange.get());
+    }
+
+    if (boost::optional<MaterialPropertyPhaseChangeHysteresis> _phaseChangeHysteresis = modelObject.materialPropertyPhaseChangeHysteresis()) {
+      translateAndMapModelObject(_phaseChangeHysteresis.get());
+    }
```

Comment:
Wrong group too. https://github.com/NREL/EnergyPlus/blob/0c9b8c4944b5f06edb000bf0d766f067830af0ae/src/EnergyPlus/HeatBalanceManager.cc#L3899

Only:

* Material
* Material:NoMass
(* Apparently the inner materials of the CFactor / FFactor constructions or something)